Assets/StreamingAssets/Images/Furniture/Vent.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Sprites>
+    <Sprite name="vent"    x="0" y="0" w="1" h="1" pixelPerUnit="64" />
```

Comment:
There should either be 1 space between name and x, or have x be aligned with the other x.